### DISCUSSION TITLE
Difference between Cloud and Self-Hosted Community Edition?

### DISCUSSION THREAD

**Discussion by parthpat12** [lobehub/lobehub#4781] [General Question] [answered]
### 📦 Environment

Docker

### 📌 Version

Latest

### 💻 Operating System

iOS

### 🌐 Browser

Edge, Safari

### 🐛 Bug Description

Is there a features comparison chart between Cloud and Self hosted version of the app? What am I missing in the self-hosted instance in docker.

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
mostly they are same. Cloud Edition currently has a better Search Plugin now but I'm moving it to Community Edition. 

If you can self-host LobeChat, just do it, we don't earn your money. Cloud Edition is aimed for these people who don't have any skill of self-hosting.

In the future, we will provider some paid features like OpenAPI, as to it's a more advanced feature to developer.
